package.json

```diff
@@ -54,55 +54,55 @@
-    "next-seo": "^5.4.0",
-    "prism-react-renderer": "^1.3.5",
-    "react": "17.0.2",
-    "react-click-away-listener": "^2.1.0",
-    "react-copy-to-clipboard": "^5.1.0",
-    "react-dom": "17.0.2",
-    "react-instantsearch-dom": "^6.23.3",
-    "react-scroll": "^1.8.7",
-    "rehype-autolink-headings": "^6.1.1",
-    "rehype-slug": "^5.0.1",
-    "remark-external-links": "^9.0.1",
-    "remark-gfm": "^3.0.1",
-    "remark-inline-links": "^6.0.1",
-    "sass": "^1.50.0",
-    "unist-util-visit": "^4.1.0"
+    "@docsearch/react": "3.3.2",
+    "@headlessui/react": "1.7.7",
+    "@next/mdx": "13.1.2",
+    "classnames": "2.3.2",
+    "compare-versions": "6.0.0-rc.1",
+    "gray-matter": "4.0.3",
+    "next": "13.1.2",
+    "next-mdx-remote": "4.2.0",
+    "next-seo": "5.15.0",
+    "prism-react-renderer": "1.3.5",
+    "react": "18.2.0",
+    "react-click-away-listener": "2.2.2",
+    "react-copy-to-clipboard": "5.1.0",
+    "react-dom": "18.2.0",
+    "react-instantsearch-dom": "6.38.3",
+    "react-scroll": "1.8.9",
+    "rehype-autolink-headings": "6.1.1",
+    "rehype-slug": "5.1.0",
+    "remark-external-links": "9.0.1",
+    "remark-gfm": "3.0.1",
+    "remark-inline-links": "6.0.1",
+    "sass": "1.57.1",
+    "unist-util-visit": "4.1.1"
   },
   "devDependencies": {
-    "esbuild": "^0.14.36",
-    "eslint": "8.13.0",
-    "eslint-config-next": "12.3.4",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-markdown": "2.2.1",
-    "eslint-plugin-mdx": "^2.0.5",
-    "eslint-plugin-react": "^7.29.4",
-    "husky": "^7.0.4",
-    "lint-staged": "^12.3.7",
-    "markdown-link-check": "^3.10.2",
-    "markdown-spellcheck": "^1.3.1",
-    "next-sitemap": "^2.5.19",
-    "npm-run-all": "^4.1.5",
-    "pkg-dir": "^6.0.1",
-    "prettier": "^2.6.2",
-    "remark-cli": "^9.0.0",
-    "remark-lint": "^8.0.0",
-    "remark-preset-lint-consistent": "^5.1.1",
-    "remark-preset-lint-recommended": "^6.1.2",
-    "remark-toc": "^8.0.1",
-    "sass": "^1.50.0",
-    "stylelint": "^14.6.1",
-    "stylelint-config-standard": "^25.0.0",
-    "windicss-webpack-plugin": "^1.6.9"
+    "esbuild": "0.16.17",
+    "eslint": "8.31.0",
+    "eslint-config-next": "13.1.2",
+    "eslint-config-prettier": "8.6.0",
+    "eslint-plugin-cypress": "2.12.1",
+    "eslint-plugin-markdown": "3.0.0",
+    "eslint-plugin-mdx": "2.0.5",
+    "eslint-plugin-react": "7.32.0",
+    "husky": "8.0.3",
+    "lint-staged": "13.1.0",
+    "markdown-link-check": "3.10.3",
+    "markdown-spellcheck": "1.3.1",
+    "next-sitemap": "3.1.45",
+    "npm-run-all": "4.1.5",
+    "pkg-dir": "7.0.0",
+    "prettier": "2.8.2",
+    "remark-cli": "11.0.0",
+    "remark-lint": "9.1.1",
+    "remark-preset-lint-consistent": "5.1.1",
+    "remark-preset-lint-recommended": "6.1.2",
+    "remark-toc": "8.0.1",
+    "sass": "1.57.1",
+    "stylelint": "14.16.1",
+    "stylelint-config-standard": "29.0.0",
+    "windicss-webpack-plugin": "1.7.7"
```

Comment:
The dependency changes in this file were  mostly performed using `ncu -u --removeRange` as recommended in https://stackoverflow.com/a/16074029

But then I manually removed the `^` prefix from some of the untouched dependencies, for consistency.